system/jlib/jsocket.cpp

```diff
@@ -7017,7 +7017,11 @@ class CWhiteListHandler : public CSimpleInterfaceOf<IWhiteListHandler>, implemen
 
         if (responseText)
         {
-            responseText->append("Access denied! [client ip=");
+            StringBuffer srvIpStr;
+            queryHostIP().getIpText(srvIpStr);
+            responseText->append("Access denied! [server ip=");
+            responseText->append(srvIpStr.str());
```

Comment:
trivial: but more efficient do away with the extra StringBuffer, and do getIpText(*responseText) here

Reply:
ok, good idea; updated.